milvus/metadata.csv

```diff
@@ -0,0 +1,387 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
+milvus.bf_search_cnt.bucket,count,,,,number of bf search per request,0,milvus,,,
+milvus.bf_search_cnt.count,count,,,,number of bf search per request,0,milvus,,,
+milvus.bf_search_cnt.sum,count,,,,number of bf search per request,0,milvus,,,
+milvus.bitset_ratio.bucket,count,,,,bitset ratio,0,milvus,,,
+milvus.bitset_ratio.count,count,,,,bitset ratio,0,milvus,,,
+milvus.bitset_ratio.sum,count,,,,bitset ratio,0,milvus,,,
+milvus.build_info,gauge,,,,Build information of milvus,0,milvus,,,
+milvus.build_latency.bucket,count,,,,index build latency (s),0,milvus,,,
+milvus.build_latency.count,count,,,,index build latency (s),0,milvus,,,
+milvus.build_latency.sum,count,,,,index build latency (s),0,milvus,,,
+milvus.cache_hit_cnt.bucket,count,,,,cache hit cnt per request,0,milvus,,,
+milvus.cache_hit_cnt.count,count,,,,cache hit cnt per request,0,milvus,,,
+milvus.cache_hit_cnt.sum,count,,,,cache hit cnt per request,0,milvus,,,
+milvus.cgo.active_future_total,gauge,,,,Total number of active futures.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.bucket,count,,second,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.count,count,,second,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.sum,count,,second,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.bucket,count,,second,,Duration of cgo call in queue.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.count,count,,second,,Duration of cgo call in queue.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.sum,count,,second,,Duration of cgo call in queue.,0,milvus,,,
+milvus.cgo.running_cgo_call_total,gauge,,,,Total number of running cgo calls.,0,milvus,,,
+milvus.datacoord.channel_checkpoint_unix_seconds,gauge,,,,channel checkpoint timestamp in unix seconds,0,milvus,,,
```

Comment:
Hmm do you know what this metric represents? If it's a time stamp then it might be better collected as a type: time_elapsed and the metric can be something like time_since_last_checkpoint or something.